README.md

```diff
@@ -33,7 +33,13 @@ e.g Enable offline form submission with following settings for the forms availab
 /contact
 /feedback
 ```
- 
+
+## Steps to setup add to home screen for your site.
+
+- Add app icon images of all sizes in your WordPress theme assets folder with `your-theme/assets/img/icon-{width x height}.png` path
+- The valid size of the images can be 72x72, 96x96, 128x128, 144x144, 152x152, 192x192, 384x384, 512x512.
+- The path of the icons can be changed with `rt_pwa_extensions_app_icon_{height}_{width}` filter
+- Example image file name `icon-192x192.png`.
```

Comment:
It would be great if you add an example with a code snippet explaining filter use.